bigtop-packages/src/rpm/bigtop-select/SPECS/bigtop-select.spec

```diff
@@ -32,16 +32,15 @@ BuildRoot:  %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
 BuildArch:  noarch
 Source0:    install_select.sh
 Source1:    LICENSE
-Source2:    conf-select
-Source3:    distro-select
+Source2:    distro-select
 Requires:   bash
 
 # "which" command is needed for a lot of projects.
 # It is part of the package "util-linux" on suse and "which" everywhere else
 %if  %{?suse_version:1}0
 Requires:  util-linux
 %else
-Requires:       which
+Requires:       which, python3
```

Comment:
At present, all Python dependencies have been removed from Bigtop, and it is now up to the user to manage them. This seems to be redundant here.